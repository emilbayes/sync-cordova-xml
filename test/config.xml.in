<?xml version='1.0' encoding='utf-8'?>
<widget version="0.0.0" xmlns="http://www.w3.org/ns/widgets">
    <name>Config</name>
    <description>
    </description>
</widget>
