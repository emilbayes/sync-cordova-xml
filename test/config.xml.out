<?xml version='1.0' encoding='utf-8'?>
<widget version="1.1.0" xmlns="http://www.w3.org/ns/widgets">
    <name>pkg</name>
    <description>pkg-desc</description>
</widget>
